pkg/util/cluster/cluster.go

```diff
@@ -113,6 +114,28 @@ func New(log *logrus.Entry, env env.Core, ci bool) (*Cluster, error) {
 		peerings:                          network.NewVirtualNetworkPeeringsClient(env.Environment(), env.SubscriptionID(), authorizer),
 	}
 
+	if ci {
+		if env.IsLocalDevelopmentMode() {
+			c.ciParentVnet = fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/virtualNetworks/dev-vnet", c.env.SubscriptionID(), c.env.ResourceGroup())
+		} else {
+			// This is dirty, but it used to be hard coded only for pub cloud.
+			// TODO pick right config value to get sub and resource group
+			if env.Environment().Name == azure.USGovernmentCloud.Name {
+				c.ciParentVnet = "/subscriptions/28015960-ee66-4844-8037-fc28b0560bf1/resourceGroups/e2einfra-usgovvirginia/providers/Microsoft.Network/virtualNetworks/dev-vnet"
+			} else {
+				// default to prior behavior, public cloud int
+				c.ciParentVnet = "/subscriptions/46626fc5-476d-41ad-8c76-2ec49c6994eb/resourceGroups/e2einfra-eastus/providers/Microsoft.Network/virtualNetworks/dev-vnet"
+			}
+		}
+
+		r, err := azure.ParseResourceID(c.ciParentVnet)
+		if err != nil {
+			return nil, err
+		}
+
+		c.ciParentVnetPeerings = network.NewVirtualNetworkPeeringsClient(env.Environment(), r.SubscriptionID, authorizer)
+	}
+
```

Comment:
This whole block was removed in https://github.com/Azure/ARO-RP/pull/1573 along with other bits that were not pulled into this PR.  Please review the changes and make sure you've got a clean rebase with only intended changes.